examples/federated_learning/mistnet/client.yml

```diff
@@ -0,0 +1,68 @@
+clients:
```

Comment:
We may need to discuss whether this yaml file needs to be generated by the developer or just for plato compatibility.

Reply:
We have a discussion with @jaypume, this Plato configuration file will be automatically generated based on the CRD.